src/twisted/test/test_twistd.py

```diff
@@ -1013,22 +903,132 @@ def test_setUidSameAsCurrentUid(self):
         If the specified UID is the same as the current UID of the process,
         then a warning is displayed.
         """
-
-        # FIXME:https://github.com/twisted/twisted/issues/10332
```

Comment:
It looks like I reported this issue some time ago.

The thing is that at the start of the test there was no failure... so the extra failure was generated while running the test.

To help troubleshoot these tests, I have updated the code to assert the warnings length last, and show all warnings on error.